web/pdf_find_controller.js

```diff
@@ -158,18 +158,19 @@ function normalize(text) {
         return replacement;
       }
 
-      if (p2) {
+      if (p3) {
```

Comment:
It feels slightly confusing that the `p{n}` cases are now appearing out of order, but I'm assuming that it's actually necessary now!?